Tests/Controller/ControllerTest.php

```diff
@@ -109,11 +109,17 @@ public static function dataProviderForTestGenerateWithCallback()
         );
     }
 
-    public function testGenerateWithInvalidCallback()
+    /**
+     * @testWith ["(function xss(x) {evil()})"]
```

Comment:
I'd rather create a data provider method than defining PHP code in comments

Reply:
or maybe even creating a separate test `testGenerateWithNonStringCallback` for the new case, as only that case needs to be tagged as `@legacy`